.github/workflows/regression-report.yml

```diff
@@ -496,6 +496,17 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_REGION }}
 
+      - name: Print config.xml
+        run: |
+          echo "Contents of config.xml:"
+          echo "-------------------------"
+          if [ -f .ci-temp/contribution/checkstyle-tester/diff_config.xml ]; then
+            cat .ci-temp/contribution/checkstyle-tester/diff_config.xml
```

Comment:
Please do this also for list of projects

Reply:
done